packages/Image/src/index.native.js.flow

```diff
@@ -44,7 +44,7 @@ declare module "constelation-image" {
       paddingHorizontal?: number,
       position?: 'absolute' | 'relative',
       ratioGrow?: boolean,
-      refNode?: () => {},
+      refNode?: (node?: Image) => void,
```

Comment:
Are we able to say it returns React Native's `Image`? I think this `Image` refers to itself (constelation's), which is not what `refNode` returns.